CONTRIBUTING.md

```diff
@@ -0,0 +1,166 @@
+> Before you start contributing, you must read and abide by our
+**[Code of Conduct](./CODE_OF_CONDUCT.md)**.
+>
+> Additionally, the Linux Foundation (LF) requires all contributions include per-commit sign-offs.
+> Ensure you use the `-s` or `--signoff` flag for every commit.
+>
+> For more details, see the [LF DCO wiki](https://wiki.linuxfoundation.org/dco)
+> or [this Pi-hole signoff guide](https://docs.pi-hole.net/guides/github/how-to-signoff/).
+
+* [Contributing code](#contributing-code)
+    * [Getting started](#getting-started)
+    * [Environment Setup](#environment-setup)
+    * [New to Node.js?](#new-to-nodejs)
+* [Contributing steps](#contributing-steps)
+* [How to run the Scorecard Monitor locally](#how-to-run-the-scorecard-monitor-locally)
+* [How to install the dependencies](#how-to-install-the-dependencies)
+* [How to run the tests](#how-to-run-the-tests)
+* [How to run the linter](#how-to-run-the-linter)
+* [PR Process](#pr-process)
+* [Changelog and versioning](#changelog-and-versioning)
+* [Releasing a new version](#releasing-a-new-version)
+* [What to do before submitting a pull request](#what-to-do-before-submitting-a-pull-request)
+* [Where the CI Tests are configured](#where-the-ci-tests-are-configured)
+* [Updating Docs](#updating-docs)
+
+## Contributing code
+
+### Getting started
+
+1.  Create [a GitHub account](https://github.com/join)
+1.  Create a
+        [personal access token](https://docs.github.com/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
+1.  Set up your [development environment](#environment-setup)
+
+### Environment Setup
+
+You must install these tools:
+
+1.  [`git`](https://help.github.com/articles/set-up-git/): For source control
+
+1.  [`node`](https://nodejs.org/en/download/package-manager): You need node version
+        `v20+`. The project includes support for [nvm](https://github.com/nvm-sh/nvm).
+
+### New to Node.js?
+
+If you're unfamiliar with Node.js, there are plenty of articles, resources, and books.
+We recommend starting with several resources from the official Node.js website:
+
+* [Introduction to Node.js](https://nodejs.org/en/learn/getting-started/introduction-to-nodejs)
+
+## Contributing steps
+
+1.  Identify an existing issue you would like to work on, or submit an issue describing your proposed change to the repo in question.
+1.  The repo owners will respond to your issue promptly.
+1.  Fork the desired repo, develop and test your code changes.
+1.  Submit a pull request.
+
+## How to run the Scorecard Monitor locally
+
+Currently, this project is consumed as a GitHub Action, so local development is quite limited. In order to test the full workflow, you can consume directly your fork and branch from another project or by adding a new workflow in your fork.
+
+Aside from this, it is possible to test certain things locally, like the `utils.js` file. Just check the test folder `_tests_/utils.test.js` to get a better idea.
```

Comment:
```suggestion
Aside from this, it is possible to test certain things locally, like the `utils.js` file. Check the test folder `_tests_/utils.test.js` to get a better idea.
```

Phrasing (avoid using phrases like "just", "simply", which may imply that things are easier than they may be for the user)